### DISCUSSION TITLE
recording which os user ran a migration?

### DISCUSSION THREAD

**Discussion by cjw296** [sqlalchemy/alembic#1499] [Usage Questions] [answered]
Is it possible to record, in the migrations table, which operating system user ran an alembic migration?
(ie: not which database user, although I don't know how to do that either: that'll just be the ddl admin user credentials)

**Comment by zzzeek** [comment, marked as answer]
in a different table, yes.  check out this repo for an example: https://github.com/jpassaro/Audit-Alembic
